packages/at_persistence_secondary_server/lib/src/log/commitlog/at_commit_log.dart

```diff
@@ -220,4 +202,14 @@ class AtCommitLog implements AtLogType {
   void removeEventListener(AtChangeEventListener atChangeEventListener) {
     _atChangeEventListener.remove(atChangeEventListener);
   }
+
+  /// Returns the list of commit entries greater than [sequenceNumber]
+  /// throws [DataStoreException] if there is an exception getting the commit entries
+  Future<List<CommitEntry>> getChanges(int? sequenceNumber, String? regex,
+      {int? limit});
+
+  /// Return the latest [CommitEntry] for the given key.
+  ///
+  /// If CommitEntry does not exist for the given key, [NullCommitEntry] is returned.
+  FutureOr<CommitEntry?> getLatestCommitEntry(String key);
```

Comment:
Changing method signature is a breaking change. Why have you changed from `CommitEntry? getLatestCommitEntry(String key)`?

Reply:
In order to fetch the latest CommitEntry, the implementation in `at_client_commit_log_keystore` is an async method. So added `FutureOr` to the return type. 

Also, currently, this method is only invoked from the server side, where the method signature is unchanged:

Snippet from  
"packages/at_persistence_secondary_server/lib/src/log/commitlog/server/at_server_commit_log_keystore.dart"

```dart
CommitEntry getLatestCommitEntry(String key) {
```

Reply:
Got it. OK:
(1) It's still a breaking change and we should make every effort to avoid those
(2) The client-side implementation will be the same because we will maintain the in-memory map there as well. I'll update your branch with my changes

Reply:
Sure Gary, Thank you.